internal/util/openapi/augments/builtin.go

```diff
@@ -0,0 +1,105 @@
+package augments
+
+const JsonPatchBuiltin = `
+[
+  {"op": "add",
+    "path": "/definitions/io.k8s.apimachinery.pkg.apis.meta.v1.ObjectMeta/properties/annotations/x-kubernetes-field-meaning",
```

Comment:
It seems we need to add these kind of augment for every GVK.
Doing it manually doesn't seems to be good, since human may accidentally forget to do it for some types.
We may want to have a small tool to do these cross-cutting changes (e.g. .metadata.annotations, .metadata.labels and .metadata.namespace) for us eventually.

Reply:
Agreed, a tool for cross-cutting changes will be better than doing it manually.

Reply:
A note: for anything in metadata, we only need to add it once. There is one definition for ObjectMeta referenced by all GVKs.